ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -44,9 +49,36 @@ export default class StatusDropdown extends React.Component {
       showNotesModal: false,
       showTagsModal: false,
       selectedValue: this.props.issueTrackers[0].text,
+      jobList: [],
     };
   }
 
+  findRelevantAlert = () => {
```

Comment:
Instead of looking for a single relevant alerts, let's look for all regression alerts.
It's imperative to provide this data & also stress the frontend to see how well it behaves with such calculus.

Reply:
Until we don't provide links for all regression items, I don't see this stress necessary.
But I see your point, though.

Reply:
There's a back & forth here with what gets implemented & what not.
Usually, these kind of issues are mitigated with acceptance criteria.

Please provide them as such, so we know what to expect from this PR.

Reply:
aye